braininventory/get.py

```diff
@@ -100,3 +100,20 @@ def report():
 
 	return report
 
+#The following block is a function that finds the number of rows that have 'true' under the key 'exists'.
+def __get_exists_true(df):
+  return len(df[df['exists']== True]) #The true listed in the dataframe is the Boolean true value.
+exists_true = __get_exists_true(df) #This line assigns the returned value to a variable and print the value using the variable name.
+print(exists_true)
+
+#The following block is a function that finds the number of total rows.
+def __get_exists_total(df):
+  return len(df) #len counts the number of rows in the dataframe.
+exists_total = __get_exists_total(df) #The value of the total number of rows is now stored to the variable exists_total.
+print (exists_total)
+
+#Now that we have the total number of exists and the total number of rows in the dataframe we can find the fraction of the total that exist using simple division.
+proportion = exists_true/exists_total 
```

Comment:
Wrap this in a function like you did the others, please. 

Good work. 
![](https://media2.giphy.com/media/j6wQILdEcwUqdy6MJH/giphy.gif?cid=ecf05e47koe99v14lacztkbmu6cumkexdkv9ajkgiqwdgcmr&ep=v1_gifs_search&rid=giphy.gif&ct=g)